tasks/retry_strategy.py

```
@@ -0,0 +1,398 @@
+        
+    Returns:
+        リトライ機能付きの関数
+    """
+    def wrapper(self: Task, *args, **kwargs):
+        task_id = self.request.id
+        retry_count = self.request.retries
+        session_id = args[0] if args else kwargs.get('session_id', 'unknown')
+        channel = f"stream:{session_id}"
+        
+        # 部分レスポンス管理
+        partial_chunks = []
```

Ruff (0.12.2)

Local variable `partial_chunks` is assigned to but never used

Remove assignment to unused variable `partial_chunks`

(F841)